ingestors/kafka/src/main/scala/hydra/kafka/model/TopicMetadataV2Transport.scala

```diff
@@ -95,27 +105,27 @@ final case class TopicMetadataV2Request(
 }
 
 object TopicMetadataV2Request {
+
   type NumPartitionsPredicate = Greater[W.`9`.T] And Less[W.`51`.T]
   type NumPartitions = Int Refined NumPartitionsPredicate
+
   object NumPartitions extends RefinedTypeOps[NumPartitions, Int]
-  type SubjectRegex = MatchesRegex[W.`"""^(?:skills|flow|tech|fin|dvs|_[a-zA-Z0-9]+)\\.[a-zA-Z0-9\\-\\.]+"""`.T]
+
+  val AllowedOrganizations: String = "cloud|skills|flow|tech|fin|dvs|_[a-zA-Z0-9]+"
+
+  val regex: String = s"^(?=^.{0,255}$$)(?:$AllowedOrganizations)(\\.[a-zA-Z0-9]+(\\-[a-zA-Z0-9]+)*)+"
```

Comment:
Our limit on topic length is 249 characters.

Reply:
changed